src/main/java/ch/njol/skript/Skript.java

```diff
@@ -641,12 +641,12 @@ public void run() {
 					exception(e, "Could not load aliases concurrently");
 				}
 
-				if (TestMode.ENABLED) {
+				if (TestMode.ENABLED || TestMode.JUNIT) {
```

Comment:
junit should only be true if test mode is already enabled.

Reply:
Not sure if you mean to ensure both are true with && or if it's redundant
```suggestion
				if (TestMode.ENABLED) {
```

Reply:
Redundant